sklearn/linear_model/coordinate_descent.py

```diff
@@ -132,17 +132,20 @@ def lasso_path(X, y, eps=1e-3, n_alphas=100, alphas=None,
 
     The Lasso optimization function varies for mono and multi-outputs.
 
-    For mono-output tasks it is::
+    For mono-output tasks it is
 
-        (1 / (2 * n_samples)) * ||y - Xw||^2_2 + alpha * ||w||_1
+    .. math::
+        \\frac{1}{2 * \\mathrm{n\\_samples}} * ||y - Xw||^2_2 + \\alpha * ||w||_1
 
-    For multi-output tasks it is::
+    For multi-output tasks it is
 
-        (1 / (2 * n_samples)) * ||Y - XW||^2_Fro + alpha * ||W||_21
+    .. math::
+        \\frac{1}{2 * \\mathrm{n\\_samples}} * ||Y - XW||^2_{Fro} + \\alpha * ||W||_{21}
```

Comment:
Fro should be mathrm.

21 should not look like 21: you can check up on notations for this, but 2 and 1 need to be separated, if only by a space, indicative of using l2 norm on one axis and l1 on the other.